v2/emailpassword/custom-ui/handling-session-tokens.mdx

```diff
@@ -66,6 +71,10 @@ class MainApplication: Application() {
 }
 ```
 
+:::info
+Note: When making network requests you do not need to call `HttpURLConnection.connect` because SuperTokens does this for you.
+:::
```

Comment:
```suggestion
:::note
When making network requests you do not need to call `HttpURLConnection.connect` because SuperTokens does this for you.
:::
```